packages/world/ts/config/v2/input.ts

```diff
@@ -87,12 +87,16 @@ export type WorldInput = evaluate<
      * as public systems at `namespace/ContractName`, unless overridden
      *
      * The key is the system name (capitalized).
-     * The value is a SystemConfig object.
+     * The value is a SystemInput object.
      */
     systems?: SystemsInput;
-    /** System names to exclude from automatic deployment */
+    /**
+     * System names to exclude from automatic deployment
+     * @deprecated
```

Comment:
```suggestion
```
let's not deprecate this yet - I am not sure the exact approach we're taking since most things in `SystemInput` are deploy related and unclear if its worth moving them all under a `deploy` key